test/suite/test_prepare_hs04.py

```diff
@@ -41,7 +41,7 @@ def timestamp_str(t):
 #
 class test_prepare_hs04(wttest.WiredTigerTestCase):
     # Force a small cache.
-    conn_config = 'cache_size=50MB,statistics=(fast)'
+    conn_config = 'cache_size=5MB,statistics=(fast)'
```

Comment:
I know you said you reduced the cache to repro. Did you intend to keep this change permanently?

Reply:
Yes I want to make it permanent. It gives a chance to eviction to write pages with prepared updates to disk.